taipy/core/notification/event.py

```diff
@@ -62,23 +62,20 @@ class EventEntityType(_ReprEnum):
     JOB = 6
     SUBMISSION = 7
 
-
-_NO_ATTRIBUTE_NAME_OPERATIONS = {EventOperation.CREATION, EventOperation.DELETION, EventOperation.SUBMISSION}
-_UNSUBMITTABLE_ENTITY_TYPES = (
-    EventEntityType.CYCLE,
-    EventEntityType.DATA_NODE,
-    EventEntityType.JOB,
-    EventEntityType.SUBMISSION,
-)
-_ENTITY_TO_EVENT_ENTITY_TYPE = {
-    "scenario": EventEntityType.SCENARIO,
-    "sequence": EventEntityType.SEQUENCE,
-    "task": EventEntityType.TASK,
-    "data": EventEntityType.DATA_NODE,
-    "job": EventEntityType.JOB,
-    "cycle": EventEntityType.CYCLE,
-    "submission": EventEntityType.SUBMISSION,
-}
+    @classmethod
+    def add_member(cls, name, value):
```

Comment:
EventEntityType is exposed so that our users can use it. I suspect the new dynamically added members won't have auto-completion, even if they are added by Taipy (or Taipy Enterprise) directly.
That could be a problem.

Reply:
True, there won't be autocompletion. The only way for autocompletion is by defined all member here.